packages/snaps-utils/src/errors.ts

```diff
@@ -43,7 +43,7 @@ export class WrappedSnapError extends Error {
 
     this.#error = error;
     this.#message = message;
-    this.#stack = getErrorStack(error);
+    this.#stack = getErrorStack(error) ?? undefined;
```

Comment:
Can we get into a situation where we serialize this undefined? If so, that would cause problems with JSON-RPC

Reply:
`.toJSON` removes it. I added a test case for this.

The `?? undefined` was necessary to match the previous behaviour, since I changed `getErrorStack` to return `null` in this PR.

Reply:
Can you point me to that test-case?

Reply:
https://github.com/MetaMask/snaps/blob/34b7cf006c700ebed65efddf2e33ae58d9f8d096/packages/snaps-utils/src/errors.test.ts#L40-L60

and

https://github.com/MetaMask/snaps/blob/34b7cf006c700ebed65efddf2e33ae58d9f8d096/packages/snaps-utils/src/errors.test.ts#L303-L314